.github/workflows/python-ci.yml

```
@@ -25,18 +22,16 @@ jobs:
       uses: actions/setup-python@v3
```

actionlint (1.7.9)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)